src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -46,7 +46,8 @@ class Executor {
 };
 
 // Define a void GCS RPC client method.
-#define VOID_GCS_RPC_CLIENT_METHOD(SERVICE, METHOD, grpc_client, SPECS)                \
+#define VOID_GCS_RPC_CLIENT_METHOD(SERVICE, METHOD, grpc_client, method_timeout_ms,    \
```

Comment:
where is `method_timeout_ms` used in this macro? Seems you hard-coded it as -1 below?